controls/cf_serverd.cf

```diff
@@ -38,19 +38,10 @@ body server control
       denybadclocks         => "false";
 
     windows::
-      cfruncommand => "$(sys.cf_agent) -f \"$(sys.update_policy_path)\" & $(sys.cf_agent)";
+      cfruncommand => "$(sys.cf_agent) -I -f \"$(sys.update_policy_path)\"  &  $(sys.cf_agent) -I -D cfruncommand";
 
-    hpux::
-      cfruncommand => "$(def.cf_runagent_shell) -c \"$(sys.cf_agent) -f $(sys.update_policy_path)\" ; $(sys.cf_agent)";
-
-    aix::
-      cfruncommand => "$(def.cf_runagent_shell) -c \"$(sys.cf_agent) -f $(sys.update_policy_path)\" ; $(sys.cf_agent)";
-
-    solaris::
-      cfruncommand => "$(def.cf_runagent_shell) -c \"$(sys.cf_agent) -f $(sys.update_policy_path)\" ; $(sys.cf_agent)"; 
-
-    !(windows|hpux|aix|solaris)::
-      cfruncommand => "$(def.cf_runagent_shell) -c \"$(sys.cf_agent) -f $(sys.update_policy_path)\" ; $(sys.cf_agent)";
+    !windows::
+      cfruncommand => "$(def.cf_runagent_shell) -c \'$(sys.cf_agent) -I -f $(sys.update_policy_path)  ;  $(sys.cf_agent) -I -D cfruncommand\'";
```

Comment:
Oh lets be sure to define `cfruncommand` class during the update policy as well (windows too)

I would probably switch to using the long options for readability. Other than that seems OK to me (though I use runagent almost never, so I might be overlooking things).